TLM/TLM/UI/SubTools/SpeedLimits/Overlay/SpeedLimitsOverlay.cs

```diff
@@ -400,20 +412,14 @@ private struct CachedSegment {
         /// <param name="netManager">Access to map data.</param>
         /// <param name="camPos">Camera position to consider.</param>
         /// <param name="speedLimitManager">Query if a segment is eligible for speed limits.</param>
-        private void ShowSigns_RefreshVisibleSegmentsCache(NetManager netManager,
-                                                           Vector3 camPos,
-                                                           SpeedLimitManager speedLimitManager) {
-            // cache visible segments
+        private void ShowSigns_RefreshVisibleSegmentsCache(Vector3 camPos) {
+
             this.cachedVisibleSegmentIds_.Clear();
             this.segmentCenters_.Clear();
 
             for (uint segmentId = 1; segmentId < NetManager.MAX_SEGMENT_COUNT; ++segmentId) {
-                ref NetSegment segment = ref ((ushort)segmentId).ToSegment();
 
-                // Ignore: Bad segments
-                if (!segment.IsValid()) {
```

Comment:
Removed this because it's already included in `segment.MayHaveCustomSpeedLimits()`